sound/soc/sof/core.c

```diff
@@ -331,6 +332,13 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	}
 
 	/* boot the firmware */
+
+	/*
+	 * record the jiffies of bootup, send the time stamp between resume
+	 * and bootup can avoid overflow when DSP calculate ms to ticks if
+	 * the time stamp of bootup is a huge number.
```

Comment:
Sorry, I did not understand the comment here. My understanding was that the firmware only count timer ticks since it boots so isn't even aware of any offset and the number of resumes.

Reply:
@plbossart Here we record the time stamp of first firmware boot. It should be closed to the time stamp of system boot. If we don't do this and use the absolute jiffies, we could get a huge value. I think what we need is the time stamp of system boot up instead of a time stamp since long time age. Maybe there is a more suitable value than jiffies, I will check it latter this week.